File: HFY.Entities/Concrete/Araclar.cs
using HFY.Core.Entities;
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.Text;

namespace HFY.Entities.Concrete
{
    public class Araclar : IEntity
    {
        [Key]
        public int AracID { get; set; }

        public string Plaka { get; set; }

        public int MarkaID { get; set; }

        public int ModelID { get; set; }

        public int FirmaID { get; set; }

        public byte Aks1 { get; set; }

        public byte Aks2 { get; set; }

        public byte Aks3 { get; set; }

        public byte Aks4 { get; set; }

        public int OlusturanId { get; set; }

        public DateTime OlusturmaTarihi { get; set; }

        public int DuzenleyenId { get; set; }

        public DateTime DuzenlemeTarihi { get; set; }

        public bool Aktif { get; set; }

        public bool ListeAktiflik { get; set; }
    }
}

File: HFY.MvcWebUI.Angular/ClientApp/src/app/lazy-general-page/classes/araclar/araclar.ts
export class Araclar {
    AracID: number;
    Plaka: string;
    MarkaID: number;
    ModelID: number;
    FirmaID: number;
    Aks1:number;
    Aks2:number;
    Aks3:number;
    Aks4:number;
    OlusturanId: number;
    OlusturmaTarihi: Date | string;
    DuzenleyenId: number;
    DuzenlemeTarihi: Date | string;
    Aktif: boolean;
    ListeAktiflik: boolean;

    jqueryDataTableAraclar() {
        return [{ data: 'FirmaID' }, { data: 'Plaka' }, { data: 'MarkaID' }, { data: 'ModelID' }, { data: 'ModelID' }, // araç kategorisi
        { data: 'Aktif' } // İşlemler
        ]
    }
}
